Pcap++/header/PcapLiveDeviceList.h

```diff
@@ -1,12 +1,16 @@
 #pragma once
 
+#include "DeprecationUtils.h"
 #include "IpAddress.h"
 #include "PcapLiveDevice.h"
 #include <vector>
+#include <memory>
 
 
 /// @file
 
+#define PCPP_DEPRECATED_RAW_PTR_API PCPP_DEPRECATED("This method is deprecated in favor of the SmartPtrAPI overload.")
```

Comment:
Doxygen generation really dislikes this macro for some reason, when used on a member function. No idea why... tried several things to fix them including mirroring the way the `define` is written in `TcpLayer.h`, but none of them worked. Would like a second opinion on why the generation fails on the macro.

Reply:
Doxygen test fails with the following:
```
Preprocessing /tmp/cirrus-ci-build/Pcap++/head/tmp/cirrus-ci-build/Pcap++/header/PcapLiveDeviceList.h:62: error: Found ';' while parsing initializer list! (doxygen could be confused by a macro call without semicolon) (warning treated as error, aborting now)
er/MBufRawPacket.h...
```